workers_launch_template.tf

```diff
@@ -97,6 +97,11 @@ resource "aws_autoscaling_group" "workers_launch_template" {
     "health_check_grace_period",
     local.workers_group_defaults["health_check_grace_period"]
   )
+  capacity_rebalance = lookup(
```

Comment:
Can you please do the same for ASG for workers (Launch Configuration). See workers.tf.

Reply:
done